rust-toolchain.toml

```diff
@@ -16,5 +16,5 @@
 # under the License.
 
 [toolchain]
-channel = "nightly-2023-08-28"
```

Comment:
Try search `nightly-2023-08-28` in our codebase, I remember some CI files also hard code it.

Reply:
OK